README.md

```diff
@@ -112,14 +114,20 @@ Output | Type | Description
  
  cfMedipsQC is designed to produce several QC metrics using samtools and picard. Runs it's own alignment with bowtie2
  
-### Run read trimming with trimmomatic
+###Calculate scaling coefficient for dynamic allocation of RAM
```

Comment:
Markdown headers will not display properly without a space between `###` and the first word. Ditto for line 123

Reply:
You are most likely right, yet it renders fine in bash. However, Github does not show it correctly in edit mode